dms/models/dms_access_groups.py

```diff
@@ -1,17 +1,27 @@
 # Copyright 2017-2019 MuK IT GmbH
 # Copyright 2020 RGB Consulting
+# Copyright 2024 Timothée Vannier - Subteno (https://www.subteno.com).
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
 
 from odoo import _, api, fields, models
 from odoo.exceptions import ValidationError
 
 
 class DmsAccessGroups(models.Model):
+    # ----------------------------------------------------------
```

Comment:
This comment is totally unneeded. They are clear and put at the beginning without empty lines in the middle, so they are very recognizable.